src/screens/EncounterUserDetailScreen.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ScrollView } from 'react-native';
+
+import { EncounterUserDetails, SectionHeader, WrapperWithOrientation } from '../components';
+import { texts } from '../config';
+
+// TODO: accesibility labels
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const EncounterUserDetailScreen = ({ route }: any) => {
+  return (
+    <ScrollView>
+      {/* TODO: soll hier tatsächlich gender logic hin? */}
```

Comment:
i think we/the process do not need a section header here (and in other places as well), right?

Reply:
i am not sure. currently the navigation screen title is constant, and the section header always gives more info on what the screen actually is.

i think it is fine this way, but we should probably choose a gender agnostic `SectionHeader`

Reply:
I added it as a task to the ticket now. i will resolve this conversation for now